vm/systemSmartContracts/delegation.go

```diff
@@ -207,7 +207,7 @@ func (d *delegation) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCo
 		return d.getUserUnStakedValue(args)
 	case "getUserUnBondable":
 		return d.getUserUnBondable(args)
-	case "getUserUnDelegatedList":
+	case "getUserUndelegatedList":
```

Comment:
was this change necessary? other methods are unDelegate, unBond, unStake, unJail

Reply:
ok